docs/content/nodes/access-api.mdx

```diff
@@ -13,21 +13,25 @@ A language-agnostic specification for this API is defined using [Protocol Buffer
 
 The Access Nodes hosted by DapperLabs are accessible at:
 
-#### Current Mainnet
-`access.mainnet.nodes.onflow.org:9000`
+
+| Network     | GRPC                                      | Web GRPC                          | REST                      |
+| -------     | ----------------------------------------- | --------------------------------- | ------------------------- | 
+| Mainnet     | `access.mainnet.nodes.onflow.org:9000`    | `mainnet.onflow.org`              | `rest-mainnet.onflow.org` |
```

Comment:
@sjonpaulbrown do you know the web gRPC endpoints?

Reply:
We will talk next week about this and sync on which host names we need to support. This will not go live until then.